### PR TITLE
Update config reference dump

### PR DESCRIPTION
**Pull request by ausi**
This is a dump of Contao 4.13 with the new configurations of 5.1 added.

### CODE REVIEW

**Review by fritzmg — APPROVED** [review]

**Comment by fritzmg** [comment]
Thank you @ausi :)

File: docs/dev/reference/config.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ Contao's configuration options can be reviewed by running the following command:
 $ vendor/bin/contao-console config:dump-reference contao
 ```
 
+The following is a dump of the configuration reference from Contao **5.1**.
```
Comment (ausi) [inline docs/dev/reference/config.md:15]: @fritzmg this is not completely accurate as it also includes configuration keys that exist in 4.13 only (like `legacy_routing` for example)
But I thought as a reference having all configurations from 4.13 and 5.1 would be best for the docs.

Comment (fritzmg) [reply docs/dev/reference/config.md:15]: Oh, so you merged them together? We could also make tabs for Contao 4(.13) and 5(.1). That's easier to maintain.

Comment (ausi) [reply docs/dev/reference/config.md:15]: > so you merged them together?

Yes.

> We could also make tabs for Contao 4(.13) and 5(.1).

:+1: I will create a new PR for that.

Comment (ausi) [reply docs/dev/reference/config.md:15]: Done #1191

**Pull request merged**